model/concourse.py

```diff
@@ -70,6 +70,9 @@ def image_pull_secret(self):
     def tls_secret_name(self):
         return self.raw.get('tls_secret_name')
 
+    def webhook_dispatcher(self):
```

Comment:
add to required attributes, please

Reply:
also: why does concourse "own" the dispatcher? should it not be the other way around? the dispatcher will interact (based upon concouse job mappings) with concourse instance(s) and github repositories